frontend/.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": [ "es2015" ],
   "plugins": [
+      "transform-object-rest-spread",
```

Comment:
I'm not saying no to this particular plugin, but let's try to keep them to a minimum. They do offer awesome things, yes, but let's also keep in mind that they simply transpile code into native JS, with varying levels of performance

Reply:
I agree that we should consider before using plugins that end up using heavy polyfills. But for all others that just end up transpiling to native code I disagree. The speed and complexity it sometimes removes is greater than the performance loss, to me it smells like premature optimization.

Reply:
very true, we need to keep a good balance of both, taking into account all pros and cons :+1: